pom.xml

```
@@ -70,6 +70,12 @@
 					<images>
 						<image>
 							<name>deviceinsight/k8s-secret-provisioner</name>
+							<authConfig>
+								<push>
+								   <username>${DOCKER_HUB_USER}</username>
```

Indentation mixes tabs and spaces.

```suggestion
									<username>${DOCKER_HUB_USER}</username>
```